### PR TITLE
add doravota

### PR DESCRIPTION
**Pull request by tomas-dora**
## Description

https://github.com/cosmos/chain-registry/tree/master/doravota

### Adding Chains

If adding a new chain, please ensure the following:
- [x] Chain is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).

  Note:
   - Chain's registration must have `staking` defined, with at least one `staking_token` denom specified.
   - Chain's registration must have `fees` defined; at least one fee token has low, average, and high gas prices defined.
- [x] Add chain to bottom of `zone_chains.json`

  Note:
   - RPC and REST must not have any CORS blocking of the Osmosis domain.
   - RCP node must have WSS enabled.

### Adding Assets

https://github.com/cosmos/chain-registry/blob/master/_IBC/doravota-osmosis.json

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.

  Note:
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_main` defaults to `false` (or else cite the listing rule that justifies enlisting to the Main app)

### On-chain liquidity

If adding a new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] The submitting team will ensure a pool will be created soon. Please hold off until further communication, which will notify of the Pool ID. (Please set the PR's status to Draft until the pool has been created)

OR
- [ ] A preview link is requested for the submitting team to be able to create a pool using the Osmosis Frontend.

OR
- [ ] The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### Validation Testing

If adding or updating a chain or asset, the changes must be validated from the updated build of the Osmosis Frontend. Testers will need a small amount of tokens to validate. 
  - [ ] Validaters can buy a small amount of this asset from Pool ID: {PROVIDE POOL ID}.
  - [ ] Preview build has been validated.

### CODE REVIEW

File: osmosis-1/osmosis.zone_assets.json [hunk 1]
```diff
@@ -1553,6 +1553,12 @@
       "base_denom": "erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "path": "transfer/channel-204/erc20/0x655ecB57432CC1370f65e5dc2309588b71b473A9",
       "osmosis_verified": false
+    },
+    {
+      "chain_name": "doravota",
+      "base_denom": "peaka",
+      "path": "transfer/channel-2694/peaka",
+      "osmosis_verified": true
```
Comment (JeremyParish69) [inline osmosis-1/osmosis.zone_assets.json:1561]: Please default osmosis_verified to false

**Comment by JeremyParish69** [comment]
Thanks @tomas-dora,
what is the plan for on-chain liquidity. Is there a pool id we can use?

**Comment by tomas-dora** [comment]
> Thanks @tomas-dora, what is the plan for on-chain liquidity. Is there a pool id we can use?

We are planning to create pools for osmosis and doravota on osmosis. Do we need to create the pool first?

**Comment by tomas-dora** [comment]
When we created the pool, we found out that there is a problem with token accuracy on osmosis regarding our assets, so do we need to wait for the assetlist to merge before creating the pool?

our denom: `peaka`
exponent: `18`
https://github.com/cosmos/chain-registry/blob/master/doravota/assetlist.json#L14

image

**Comment by JeremyParish69** [comment]
> When we created the pool, we found out that there is a problem with token accuracy on osmosis regarding our assets, so do we need to wait for the assetlist to merge before creating the pool?
> 
> our denom: `peaka` exponent: `18` https://github.com/cosmos/chain-registry/blob/master/doravota/assetlist.json#L14
> 

No, you do not need to wait for that display preference to be resolved before creating the pool. You can still create the pool using the frontend, but you should be mindful to use the base precision (so add 18 zeros or decimal places to whatever you would be adding)

At the chain registry, the display denom is already specified as 'DORA', which is defined to have exp 18, (technically what you have in this PR is incorrect, but it will automatically be overwritten anyway, so no need to worry about it). But the precision issue will be resolved once it's added by us to the frontend.

**Comment by tomas-dora** [comment]
@JeremyParish69  So I need to create a pool in the frontend first and then provide you the pool id for this pr to be merged?
image

**Comment by tomas-dora** [comment]
## Description

We created the pool on osmosis (pool id is `1235`) https://app.osmosis.zone/pool/1235

create pool tx detail: https://www.mintscan.io/osmosis/tx/E5720A85F1EAA742CCAE578B289D17F22B0601CB04AB68B6D55C3946705E0911

## Question

Regarding the precision issue, there is a precision issue when creating pools and appending DORA tokens, which causes me to hit a snag when add liquidity.

Untitled

Is it possible to fix the precision first (I understand merge this pr first) and we can then proceed to tweak the pool's parameter information. @JeremyParish69

**Comment by tomas-dora** [comment]
@JeremyParish69 But osmosis's front-end application seems to limit the number of inputs, so when I enter 1000000000000000000 (1DORA), the front-end reports an error.

image

File: osmosis-1/osmosis-1.chainlist.json [hunk 2]
```diff
@@ -5928,6 +5928,62 @@
...
+        "bech32PrefixConsAddr": "doravalcons",
+        "bech32PrefixConsPub": "doravalconspub"
+      },
+      "slip44": 118,
+      "fees": {
+        "fee_tokens": [
+          {
+            "denom": "peaka",
+            "fixed_min_gas_price": 100000000000,
+            "low_gas_price": 100000000000,
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.chainlist.json:5953]: Can we please add these gas prices to the chain registry?

**Comment by JeremyParish69** [comment]
> @JeremyParish69 But osmosis's front-end application seems to limit the number of inputs, so when I enter 1000000000000000000 (1DORA), the front-end reports an error.
> 
> image

Apologies, I didn't know it did that. You may use this preview link to create the pool:
https://osmosis-frontend-git-jp69-asset-dora-add-osmo-labs.vercel.app/

**Comment by tomas-dora** [comment]
@JeremyParish69 The temporary front end still has accuracy issues.

image

**Comment by JeremyParish69** [comment]
> @JeremyParish69 The temporary front end still has accuracy issues.
> 
> image

My mistake, I accidentally left the exponent at 6 instead of 18, which is why we see 12 extra zeros there. Should be fixed now.

**Comment by tomas-dora** [comment]
What's the difference here? It seems to me that I can only create the first two types of pools, and I can't create a `Supercharged pool`.

image

**Comment by tomas-dora** [comment]
> If going with a classic pool, please create a 50/50 weighted pool.

Can I change the weight now, this is the weight data from my previous pool creation.

`uosmo:ibc/dora_denom_peaka = 50000:950000`


image

**Comment by tomas-dora** [comment]
@JeremyParish69 yeah, we created
dora/osmo https://osmosis-frontend-git-jp69-asset-dora-add-osmo-labs.vercel.app/pool/1239
atom/dora https://osmosis-frontend-git-jp69-asset-dora-add-osmo-labs.vercel.app/pool/1238

**Comment by tomas-dora** [comment]
@JeremyParish69 Can you help me add DORA to swap here?

image

**Comment by JeremyParish69** [comment]
thnaks @tomas-dora , I can use one of these new pools for pricing.
No, there's no way to adjust the pool weighting (of 1235) after it's been created.
The frontend only supports creation of Weighted and StableSwap pools. Other pool types can only be created via osmosisd CLI.
DORA would show up on the Swap page once it's merged to the frontend repo--I'm just validating the build now, but it's looking like the IBC transfers are failing at the moment, so that is blocking the launch on Osmosis.

**Comment by tomas-dora** [comment]
@JeremyParish69 Yes, we ditched the `1235`. 

I checked, it was because the ibc light client wasn't updated in time, it's back up now, thanks for the feedback. You can try again and add our chain.

**Pull request merged**